### DISCUSSION TITLE
Open source contribution

### DISCUSSION THREAD

**Discussion by ritikrawat2447** [android/nowinandroid#1089] [General]
As I dive into contributing to open-source projects, I'm eager to engage with this particular project. I'm curious whether it's advisable to contribute by cloning from the main branch or from the latest active branch. Can you provide guidance on the recommended approach for contributing effectively to the project?

**Comment by SimonMarquis** [comment]
Hi @ritikrawat2447, this question has already been asked and answered in https://github.com/android/nowinandroid/discussions/260
